src/legacy/ui/ui_render/bootstrap/startup.js.hbs

```diff
@@ -7,16 +7,41 @@ try {
   window.console.log(error);
 }
 
-var useBrowserColorScheme = window.localStorage.getItem('useBrowserColorScheme') === 'true' || false;
+/** @type {'browser' | 'light' | 'dark' | null} */
+var urlThemeParam = null;
+try {
+  urlThemeParam = new URLSearchParams(window.location.search).get('theme');
```

Comment:
if this parameter only impacts dark/light mode, can we name it after that and not use `theme`?

Reply:
yes it's like `'colorscheme'` vs `'background'`, and "theme" only impacts background. i thought about something like darkMode, but it also needs to support "browser". do you have any recommendation?